src/javascript-extract.js

```diff
@@ -24,12 +24,15 @@ function getGettextEntriesFromJavaScript(argTokens = []) {
   for (let i = 0; i < allTokens.length; i = i + 1) {
     let token = allTokens[i];
     for (let gettextFunc in DEFAULT_VUE_GETTEXT_FUNCTIONS) {
+      if (!DEFAULT_VUE_GETTEXT_FUNCTIONS.hasOwnProperty(gettextFunc)) {
+        continue;
+      }
+      let args = DEFAULT_VUE_GETTEXT_FUNCTIONS[gettextFunc];
```

Comment:
I don't understand this change, can you maybe place a comment to explain why it's so ?

Reply:
I should have paid closer attention when rebasing. I tried to fix a problem (`guard-for-in` eslint violation) that has been solved in the `added babel parser support for javascript parsing` PR. I'll remove it.